app/controllers/allowances/AllowancesStartController.scala

```diff
@@ -40,29 +41,27 @@ class AllowancesStartController @Inject() (
   view: AllowancesStartView,
   businessService: BusinessService
 )(implicit ec: ExecutionContext)
-    extends FrontendBaseController with I18nSupport {
+    extends FrontendBaseController with I18nSupport with BusinessServiceLike {
 
   def onPageLoad(taxYear: Int, propertyType: PropertyType): Action[AnyContent] =
     (identify andThen getData andThen requireData).async { implicit request =>
-          val hc = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
-          businessService.getUkPropertyDetails(request.user.nino, request.user.mtditid)(hc).flatMap {
-            case Right(Some(propertyData)) =>
-              Future.successful(
-                Ok(
-                  view(
-                    AllowancesStartPage(
-                      taxYear,
-                      request.user.isAgentMessageKey,
-                      propertyData.accrualsOrCash.get,
-                      request.userAnswers,
-                      propertyType
-                    )
-                  )
+      val hc = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
```

Comment:
Line: 48 Rebase? This line already removed in another PR

Reply:
Rebased from main already